api/v1beta1/machine_types.go

```diff
@@ -230,10 +230,32 @@ type MachineStatus struct {
 	// Conditions defines current service state of the Machine.
 	// +optional
 	Conditions Conditions `json:"conditions,omitempty"`
+
```

Comment:
Should we mention on both fields that we use these values to check if the timeouts are exceeded?